Tests/Unit/NetworkServer/BasicsStation/DownstreamSenderTests.cs

```diff
@@ -121,21 +135,26 @@ public async Task SendDownstreamAsync_Succeeds_WithValidDownlinkMessage_ClassCDe
             Assert.Contains(@"""DevEui"":""FF-FF-FF-FF-FF-FF-FF-FF"",", actualMessage, StringComparison.InvariantCulture);
             Assert.Contains(@"""dC"":2,", actualMessage, StringComparison.InvariantCulture);
             Assert.Contains(@"""pdu"":""5245465551513D3D"",", actualMessage, StringComparison.InvariantCulture);
-            Assert.Contains(@"""RX2DR"":5,", actualMessage, StringComparison.InvariantCulture);
-            Assert.Contains(@"""RX2Freq"":868500000,", actualMessage, StringComparison.InvariantCulture);
-            if (rfchHasValue)
-            {
-                Assert.Contains(@"""rctx"":1,", actualMessage, StringComparison.InvariantCulture);
-            }
-            else
-            {
-                Assert.DoesNotContain("rctx", actualMessage, StringComparison.InvariantCulture);
-            }
+            // Will select DR0 as it is the second DR.
+            Assert.Contains(@"""RX2DR"":0,", actualMessage, StringComparison.InvariantCulture);
```

Comment:
This doesn't match previous values, why?

Reply:
in the previous implementation we never differentiated between rx1dr and rx2dr, therefore before the payload was set to DR 5 on both rates. Here Rx1 is set to DR5 but rxwin2 is set to DR0 on the constructor